src/time-utils.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+const ms = require('ms')
+
+function staleDaysToDate(input) {
+  try {
+    const staleDays = Number(input)
+    if (isNaN(staleDays)) {
+      const stringToMs = ms(input)
+      return new Date().getTime() - stringToMs
```

Comment:
how many times do you think you need to write new Date().getTime() before it deserves being extracted in a variable?